src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -250,19 +248,21 @@ public void run() {
     driver.run();
   }
 
-  private void launchNode(SchedulerDriver driver, Offer offer,
-        String nodeName, List<String> taskNames, String executorName) {
+  private boolean launchNode(SchedulerDriver driver, Offer offer,
+        String nodeName, List<String> taskTypes, String executorName) {
```

Comment:
Suggestion: Would be great to see a comment somewhere about the differences/relationship between nodeName, taskType, taskIdName, executorName, taskName, and taskId.